wordpress-mobile/WordPress-FluxC-Android
# Issue/5387 Remove leftovers of OrderIdentifier usage

Part of https://github.com/woocommerce/woocommerce-android/issues/5387.

Remove leftovers of OrderIdentifier usage.

### How to test
* Run example app
* Check Order Functionality
* Check Shipping Label functionality
